### PR TITLE
extra drivers should be optionals

### PR DESCRIPTION
**Pull request by nickbalestra**
There could be simple cases where action and state driver are more than enough (see pong example)

### CODE REVIEW

File: src/createCycleMiddleware.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import {run} from '@cycle/xstream-run';
 import xs from 'xstream';
 
-export default function createCycleMiddleware(mainFn, drivers) {
+export default function createCycleMiddleware(mainFn, drivers={}) {
```
Comment (lmatteis) [inline src/createCycleMiddleware.js]: it's silly but can you add a space for the optionals? `drivers = {}` reads better than `drivers={}`

File: README.md [hunk 2]
```diff
@@ -159,6 +159,19 @@ function main(sources) {
 
 Here's an example on [how the STATE driver works](https://jsbin.com/kijucaw/7/edit?js,output).
 
+NOTE: If you want to use any other driver aside ACTION and STATE, make sure to have it installed and registered. You can do so at intantiation time, via the `createCycleMiddleware` API.
```
Comment (lmatteis) [inline README.md]: typo: instantiation

Comment (nickbalestra) [inline README.md]: Thanks!

File: src/createCycleMiddleware.js (hunk 1 above)
Comment (nickbalestra) [inline src/createCycleMiddleware.js]: Is not silly at all, I agree :)
Fixed

**Pull request merged**